File: Java/Array's/Array.java
class Array{
    public static void main(String[] args) {
        String[] arr = {"apple", "banana", "cherry"};
        for (String fruit : arr) {
            System.out.println(fruit);
        }
        //print last element
        System.out.println(arr[arr.length - 1]);
        System.out.println(arr.length);
        // arr.add("mango"); // This line will cause a compilation error
        // To add an element to an array, you need to create a new array with a larger size and copy the elements over
    }
}

File: Python/Array's/Array.py
# Create an array (list) of 5 fruits
fruits = ["Apple", "Banana", "Mango", "Orange", "Grapes"]

# Print the first fruit
print("First fruit:", fruits[0])

# Print the last fruit
print("Last fruit:", fruits[-1])

# Print the length of the array
print("Length of the array:", len(fruits))

# Print all fruits using a loop
print("All fruits:")
for fruit in fruits:
    print(fruit)
# Add a new fruit to the array
fruits.append("Pineapple")
print("After adding Pineapple:", fruits)
# Remove a fruit from the array
fruits.remove("Mango")
print("After removing Mango:", fruits)
# Sort the array of fruits alphabetically
fruits.sort()
print("Sorted fruits:", fruits)
# Reverse the array of fruits
fruits.reverse()
print("Reversed fruits:", fruits)
# Access a slice of the array (first three fruits)
first_three_fruits = fruits[:3]
print("First three fruits:", first_three_fruits)